s4e-web/cypress/integration/map/map-layers.spec.ts

```diff
@@ -1,141 +1,290 @@
 /// <reference types="Cypress" />
 
 import { Login } from '../../page-objects/auth/auth-login.po';
-import { Layers } from '../../page-objects/map/map-layers.po';
+import { MapLayers } from '../../page-objects/map/map-layers.po';
 import { GeneralModal } from '../../page-objects/modal/general-modal.po';
 import { ConfirmModal } from '../../page-objects/modal/confirm-modal.po';
 
 
-describe.skip('Map Layers', () => {
-  before(() => {
-    cy.fixture('users/zkMember.json').as('zkMember');
-    cy.fixture('layer-capability.xml').as('geoserverResponse');
-  });
+before(() => {
+  cy.fixture('users/zkMember.json').as('zkMember');
+  cy.fixture('layer-capability.xml').as('geoserverResponse');
+  cy.fixture('url-response.xml').as('urlResponse')
+});
 
-  beforeEach(function () {
-    cy.visit('/login');
-    Login
-      .loginAs(this.zkMember);
-    cy.get('[data-e2e="layers-list"] [data-e2e="picker-item-label"]').its('length').as('layerCount');
-  });
 
-  afterEach(() => {
-    Login.forceLogout();
-  });
+describe('Map MapLayers', () => {
+  context("Configuration layers for not logged in user", () => {
+    beforeEach(function () {
+      cy.visit('/login');
+    });
 
-  it('should display selected layer', () => {
-    Layers
-      .selectNthSidebarLayer(0)
-      .activeLayersCountShouldBe(1)
-      .unselectNthSidebarLayer(0)
-      .activeLayersCountShouldBe(0)
+    it('configuration layers shouldn\'t be visible without authentication', () => {
+      Login.
+        goToMapWithoutLogin();
+      MapLayers
+        .layersConfigurationAreNotVisible();
+    });
   });
 
-  it('should display all url layers', function () {
-    const label = 'Test';
-    const geoserverUrl = '/test-wms';
-
-    cy.server();
-    cy.route({
-      method: 'GET',
-      url: `${geoserverUrl}?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetCapabilities`,
-      headers: {
-        'Content-type': 'text/xml'
-      },
-      status: 200,
-      response: this.geoserverResponse
-    })
-      .as('getCapabilities');
-    Layers
-      .openManagementModal()
-      .fillForm(label, geoserverUrl);
-    cy.wait('@getCapabilities');
-
-    Layers
-      .activeLayersCountShouldBe(28)
-      .allUrlLayersCountShouldBe(28);
-
-    GeneralModal
-      .closeAndChangeContext(Layers);
-  });
+  context("Configuration layers for logged in user", () => {
```

Comment:
Wouldn't be better to call this context `Logged in user`?

Reply:
more consistent with other tests